crates/cli/Cargo.toml

```diff
@@ -20,6 +20,7 @@ serde = { workspace = true, default-features = false }
 serde_json = { workspace = true }
 javy-codegen = { path = "../codegen/", features = ["plugin_internal"] }
 javy-plugin-processing = { path = "../plugin-processing" }
+wasm-opt = { workspace = true }
```

Comment:
I think you can also delete this line. It's only used in the `plugin-processing` and `codegen` crates, there are no references in the `cli` crate.